README.md

```diff
@@ -0,0 +1,74 @@
+# Django Imager
+
+**Author:** Julien Wilson and Maelle Vance
+
+
+## Getting Started
+
+Clone this repository into whatever directory you want to work from.
+
+```bash
+$ git clone https://github.com/ellezv/django_imager.git
+```
+
+Assuming that you have access to Python 3 at the system level, start up a new virtual environment.
+
+```bash
+$ cd django_imager
+$ python3 -m venv venv
+$ source venv/bin/activate
+```
+
+Once your environment has been activated, make sure to install Django and all of this project's required packages.
+
+```bash
+(django_lender) $ pip install -r requirements.pip
+```
+
+Navigate to the project root, `imagersite`, and apply the migrations for the app.
+
+```bash
+(django_lender) $ cd imagersite
+(django_lender) $ ./manage.py migrate
+```
```

Comment:
KeyError: 'DB_USER'
 Waaaaaah!  I'm a new user and I don't know what this means!  Why, oh, why, is there no instructions regarding the use of environmental variables!  The horror!